lamindb/curators/core.py

```diff
@@ -843,7 +912,18 @@ def values(self, new_values):
     @property
     def is_validated(self) -> bool:
         """Return whether the column is validated."""
```

Comment:
There are all these occurances of the word `column` which should be replaced with `vector` because we're using this for any vector, not just columns of DataFrame (e.g. `.index` and `.columns` and presumably soon other things).